keyboards/gboards/k/buzzsaw/config.h

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2013 Oleg Kostyuk <cub.uanic@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+// Copy and worked on with love from the EZ team
+
+#pragma once
+#include "config_common.h"
+
+#define BOOTMAGIC_LITE_ROW 13
+#define BOOTMAGIC_LITE_COLUMN 2
+
+#ifndef SCROLLSTEP
+#define SCROLLSTEP 1
+#endif
+
+#define VERBOSE
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```

Comment:
This vendor ID is in use.  Please change.

Reply:
Changed in https://github.com/qmk/qmk_firmware/pull/8923/commits/15bee6e3662bb6842267d8ad4c3ac57e1e750bd0